float32[] xtrajectory
float32[] ytrajectory
string type
float32[] GlobalX
float32[] GlobalY
string useGlobalPath
string ObstacleDetected